src/pages/index.mdx

```diff
@@ -4,19 +4,73 @@ import { SiteLayout } from "../components/layout/SiteLayout";
+    <p className="flex text-center text-5xl font-display font-black text-mulberry">11th Annual oSTEM Conference 2021</p>
+    <p className="flex justify-center text-3xl font-display italic">Your seat at our table</p>
+    <hr/>
+    <p className="flex justify-center text-3xl font-display">
+    October 29 &ndash; November 1, 2021 | Virtual
+    </p>
+</div>
 
-<ConferenceCountdown />
 
-<img src="/images/conference-pic.jpg" alt="Conference Picture" style={{borderRadius: '30px' ,  padding: '1em'}}/>
 
-<p className="flex justify-center text-3xl font-display">Your seat at our table</p>
-<p className="flex justify-center text-3xl font-display">
-  October 29 &ndash; November 1, 2021, Virtual
-</p>
+<div class="grid grid-cols-5 gap-y-2 mt-9">
+    <div class="rounded-l col-span-2 tint tint-dark"><img class="object-cover w-full h-full rounded-l" src="/images/oSTEM.2019.0049_Update.png"/></div>
+    <div class="rounded-r col-span-3 bg-tint-spruce-light p-4">
+        <h3>Conference for LGBTQ+ people in STEM Fields</h3>
+        <p>
+            The oSTEM Conference returns agains this year in a virtual format.
+            This conference is a space for LGBTQ+ students and professionals in STEM fields to share their experiences and learn from one another.
+            We have programs ranging from safe spaces to talk about queer issues to discussing the latest in research conducted by our members.
+        </p>
+        <p>
+            If you want a space where you can be your authentic LGBTQ+ self and your authentic STEM self, this is the conference for you.
+        </p>
+        <div class="text-center my-4">
+            <CTAButton onClick="#">Register for the Conference</CTAButton>
+        </div>
+    </div>
+    <div class="rounded-l col-span-3 bg-tint-apricot p-4">
+        <h3>Career and Graduate School Expo</h3>
+        <p>
+            We have a dedicated Career and Graduate School Expo, allowing you to find the best opportunities from organizations and schools that will accept and support you.
+            Closer to the start of the conference, we'll be providing information about what opportunities are available from each sponsor.
+            We'll have professional development virtual events before the expo to help you get ready and make the best impression
+        </p>
+        <p>
+            If you are representing an organization that would like to exhibit at our expo, click the button below for information about the sign up process and the benefits of exhibiting at our conference.
+        </p>
+        <div class="text-center my-4">
+            <CTAButton onClick="#">Exhibitor Information</CTAButton>
+        </div>
+    </div>
+    <div class="rounded-r col-span-2 tint tint-light"><img class="object-cover w-full h-full rounded-r" src="/images/oSTEM.2019.0111.jpg"/></div>
+    <div class="rounded-l col-span-2 tint tint-dark"><img class="object-cover w-full h-full rounded-l" src="/images/oSTEM.2019.0077.jpg"/></div>
```

Comment:
I really like the hover effect but it only works on the box of the image, can we get it to trigger over the text too?

[If it's too complicated/timely we don't need it; I'm not really a coder so I have no idea the 'reasonable-ness' of this request]

Reply:
What do you mean when you want it to trigger over the text too? In that if the box the text is in is hovered over, it'll also un-tint the picture?

Reply:
Yes, if you hover over any part of the text/pic combo box the pic un-tints indicating focus.

Reply:
I *think* this is possible and something I'd like to see, but it'll take a bit more experimenting from me. I'll add it as an issue to be worked on.